README.md

```diff
@@ -28,9 +28,9 @@ func main() {
 		panic("Not in a Platform.sh Environment.")
 	}
 
-    db, err := sql.Open("mysql", p.FormattedCredentials("database", "sql_dsn"))
+  db, err := sql.Open("mysql", p.FormattedCredentials("database", "sql_dsn"))
```

Comment:
Trying this out on a template this morning shows some typos and different recommendations we should change this with:

- Typo `sql_dsn` actually registered as `sqldsn` in library
- Also, when using `db` it actually returns an `interface{}` type, not a string, so I was not able to use it to connect to `database` until I made this change:

```
func trySql(pi *psh.RuntimeConfig, w http.ResponseWriter) {

	dbString, err := pi.FormattedCredentials("database", "sqldsn")
	checkErr(err)

	db, err := sql.Open("mysql", dbString.(string))
	checkErr(err)
```

Reply:
Using the formatted credential with `sql` requires type assertion to use `dbString` as a string https://stackoverflow.com/questions/44600570/need-type-assertion-on-functions